color.go

```diff
@@ -312,6 +313,52 @@ func boolPtr(v bool) *bool {
 	return &v
 }
 
+// colorsCache is used to reduce the count of created Color objects and
+// allows to reuse already created objects with required Attribute.
+var colorsCache = make(map[Attribute]*Color)
+
+// m is mutex for locking the access to cache map.
+var m = new(sync.Mutex)
```

Comment:
This should be under `var colorsCache` and renamed to `var colorsCacheMu sync.Mutex // protects colorsCache`